BusinessLayer/BusinessLayer.pro

```diff
@@ -11,9 +11,25 @@ HEADERS += \
    BusinessContainer.h \
    CommunicationsMonitoringService/CommunicationsMonitoringService.h \
    CommunicationsMonitoringService/I_CommunicationsMonitoringService.h \
+   JsonForwarder/BatteryJsonForwarder/BatteryJsonForwarder.h \
+   JsonForwarder/BatteryJsonForwarder/I_BatteryJsonForwarder.h \
```

Comment:
The four I_*JsonForwarder interfaces are not doing anything. Since the only thing that actually uses them requires the concrete class (since it just creates them). I can't foresee any use of these interfaces so they probably could just be deleted.